### ISSUE TITLE
How many FFTs required for stable ASD loss?

### ISSUE THREAD

**Issue by alecgunny** [ML4GW/DeepClean#23]
The [`PSDLoss` loss function](../blob/main/libs/trainer/deepclean/trainer/criterion.py#L87) used to optimize DeepClean relies on a [Torch implementation](../blob/main/libs/trainer/deepclean/trainer/criterion.py#L9) of the Welch transform to estimate the PSD.

Using longer training kernels gives us more stable estimates of the PSD (or at least allows us to do this with higher frequency resolution, and I'm not sure we want worse resolution than the current default value of 1Hz), but saddles us with extra compute and memory usage/copying that we otherwise wouldn't need.

How stable do these PSD estimates need to be to get good performance with stable PSDs at test time? Can we go all the way as low as 1 and just take a single fourier transform?
